web/src/components/OverlayFooter.tsx

```diff
@@ -8,30 +9,21 @@ type OverlayFooterProps = {
 
 const OverlayFooterContainer = styled(Paper)`
   display: flex;
-  white-space: nowrap;
   padding: ${props => props.theme.spacing(2)};
-  background-color: rgb(255 255 255 / 50%);
+  background-color: ${props => props.theme.palette.surface.main}80;
+  border-radius: 0;
 
   ${props => props.theme.breakpoints.up('md')} {
     padding: 0 ${props => props.theme.spacing(1)};
   }
+
+  & .${listItemClasses.root} a {
+    height: 40px;
```

Comment:
🙃 What do you think of reducing the padding here to allow for more space?
```suggestion
    height: 40px;
    padding: 8px;
```
Otherwise it looks like this:
<img width="1084" height="86" alt="image" src="https://github.com/user-attachments/assets/1e376b8b-1044-4c81-bd96-5d9a4f10e89c" />

Reply:
This is perfect 😁